### PR TITLE
[MNO-281] Add Intercom integration

### PR DESCRIPTION
**Pull request by ouranos**

### CODE REVIEW

**Comment by BrunoChauvet** [comment]
:+1:

File: src/app/components/analytics/analytics.svc.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
+angular.module 'mnoEnterpriseAngular'
+  .service 'AnalyticsSvc', (MnoeCurrentUser, INTERCOM_ID, $window) ->
```
Comment (alexnoox) [inline src/app/components/analytics/analytics.svc.coffee]: Shouldn't it be named `Intercom` or `MnoeIntercom` ?

Comment (alexnoox) [inline src/app/components/analytics/analytics.svc.coffee]: Also the dependency order should be: `($window, MnoeCurrentUser, INTERCOM_ID)`

Order:
- Framework services
- Custom services
- Constants

It is from somewhere in this style guide: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md

**Comment by alexnoox** [comment]
LGTM

Sometime a good al' `$timeout` can be the solution.

**Pull request merged**